src/scripts/main.js

```diff
@@ -1 +1,63 @@
 'use strict';
+
+const successEl = (message) =>
+  `<div data-qa="notification" class="success">${message}</div>`;
+const errorEl = (message) =>
+  `<div data-qa="notification" class="error">${message}</div>`;
+
+const promiseHandler = (message) => {
```

Comment:
The name 'promiseHandler' is a bit vague. It's a good practice to name functions in a way that describes what they do. Consider renaming it to something more descriptive, like 'displayMessage'.